PROJECT

```diff
@@ -10,14 +14,20 @@ resources:
   controller: true
   domain: greptime.io
   kind: GreptimeDBCluster
-  path: github.com/GreptimeTeam/greptimedb-operator/apis/v1alpha1
+  path: github.com/GreptimeTeam/greptimedb-operator/api/v1alpha1
```

Comment:
Is it the right path? Maybe we should use `apis/`?

Reply:
Now is `apis`.